packages/dataviews/src/stories/fixtures.js

```diff
@@ -135,3 +135,44 @@ export const actions = [
 		callback() {},
 	},
 ];
+
+export const fields = [
+	{
+		header: 'Image',
+		id: 'image',
+		render: ( { item } ) => {
+			return (
+				<img src={ item.image } alt="" style={ { width: '100%' } } />
+			);
+		},
+		width: 50,
+		enableSorting: false,
+	},
+	{
+		header: 'Title',
+		id: 'title',
+		maxWidth: 400,
+		enableHiding: false,
+		isSearchField: true,
```

Comment:
This is a new API introduced in this PR too.

Reply:
The current name is not indicative of it's usage. Maybe `includeInGlobalSearch` ? 🤔  

I'm wondering if it's better to rely on the `type` prop with value: `string` and search all those fields, with maybe a fallback exclusion prop(`excludeFromGlobalSearch`).

Reply:
let's go with `includeInGlobalSearch` for now. I think search needs to focus on the "important fields" and not all "text fields" or exclude fields. For this reason I think opt-in is better.

Reply:
I'd follow the existing prefix nomenclature and would use something like: `enableGlobalSearch`.

Reply:
This is actually a nice thing to have, as right now, the global search is quite opaque. We should also make sure that the search is hidden if no field has `enableGlobalSearch`, etc.

Reply:
> We should also make sure that the search is hidden if no field has enableGlobalSearch, etc.

That's not something we can do because sometimes the global search uses and endpoint and might not have a direct relationship to fields.

Reply:
`enableGlobalSearch` is consistent with what we have but also weird to me. I'm going to use it but I don't like "verbs" in declarative APIs.